### DISCUSSION TITLE
CSS Reset

### DISCUSSION THREAD

**Discussion by oemueller** [SchwarzIT/onyx#318] [Ideas]
Have you considered using a modern CSS reset ? 
Like this one: https://gist.github.com/Asjas/4b0736108d56197fce0ec9068145b421

Really helps to unify styles and not having to reset margins on common elements like `p` or `h1` or `body`

Would be cool!

**Comment by larsrickert** [comment]
Do you mean for the onyx components itself? :)
We do not want to have side effects on global styles from our components so they can be used individually even together with other component libraries. So resetting styles globally is not intended for onyx itself but the project can of course do this 👍

**Reply by oemueller** [reply]
ok, yes we should not put the reset in the global scope.
It can make sense in dedicated Onyx CSS layer: https://developer.mozilla.org/en-US/docs/Web/CSS/@layer
Should be well supported by now.
Then, you can also put the grid classes into that CSS layer: packages/sit-onyx/src/styles/grid.scss,
the CSS variables and more

**Reply by JoCa96** [reply]
@oemueller Great idea, we will definitely make use of this: You can follow [#338](https://github.com/SchwarzIT/onyx/issues/338) for this.

**Comment by larsrickert** [comment]
closed with #338
